src/groundlight/client.py

```diff
@@ -172,11 +173,62 @@ def list_image_queries(self, page: int = 1, page_size: int = 10) -> PaginatedIma
             image_queries.results = [self._fixup_image_query(iq) for iq in image_queries.results]
         return image_queries
 
+    def ask_confident(
+        self,
+        detector: Union[Detector, str],
+        image: Union[str, bytes, Image.Image, BytesIO, BufferedReader, np.ndarray],
+        wait: Optional[float] = None,
+    ) -> ImageQuery:
+        """Evaluates an image with Groundlight, waiting until an answer above the confidence threshold of the detector is reached or the wait period has passed.
+        :param detector: the Detector object, or string id of a detector like `det_12345`
+        :param image: The image, in several possible formats:
+          - filename (string) of a jpeg file
+          - byte array or BytesIO or BufferedReader with jpeg bytes
+          - numpy array with values 0-255 and dimensions (H,W,3) in BGR order
+            (Note OpenCV uses BGR not RGB. `img[:, :, ::-1]` will reverse the channels)
+          - PIL Image
+          Any binary format must be JPEG-encoded already.  Any pixel format will get
+          converted to JPEG at high quality before sending to service.
+        :param wait: How long to wait (in seconds) for a confident answer.
+        """
+        return self.submit_image_query(
+            detector,
+            image,
+            wait=wait,
+        )
+
+    def ask_fast(
+        self,
+        detector: Union[Detector, str],
+        image: Union[str, bytes, Image.Image, BytesIO, BufferedReader, np.ndarray],
+        wait: Optional[float] = None,
+    ) -> ImageQuery:
+        """Evaluates an image with Groundlight, getting the first answer Groundlight can provide.
+        :param detector: the Detector object, or string id of a detector like `det_12345`
+        :param image: The image, in several possible formats:
+          - filename (string) of a jpeg file
+          - byte array or BytesIO or BufferedReader with jpeg bytes
+          - numpy array with values 0-255 and dimensions (H,W,3) in BGR order
+            (Note OpenCV uses BGR not RGB. `img[:, :, ::-1]` will reverse the channels)
+          - PIL Image
+          Any binary format must be JPEG-encoded already.  Any pixel format will get
+          converted to JPEG at high quality before sending to service.
+        :param wait: How long to wait (in seconds) for a confident answer.
+        """
+        return self.submit_image_query(
+            detector,
+            image,
+            wait=wait,
+            confidence_threshold=0.5,
```

Comment:
I don't think this is right.  Because this updates the permanent threshold for this IQ, which will cause it to never get escalated.  I think the way to do this is to set `wait=0` which causes `submit_image_query` to never go into polling mode.

The exception would be if the initial response doesn't have a real prediction yet (0.5) because the ML took too long, then this method should poll until we get something back.  I suppose we could have a `wait` timeout there.